ui/src/pages/_app.tsx

```diff
@@ -39,7 +39,7 @@ const ReactQueryDevtoolsProd = React.lazy(() =>
 // App needs to be customized in order to make MUI work with SSR
 // https://mui.com/material-ui/integrations/nextjs/#pages-router
 // https://github.com/mui/material-ui/blob/master/examples/material-ui-nextjs-pages-router-ts/pages/_app.tsx
-export default function MyApp(props: AppPropsWithLayout) {
+export default function OrcasiteApp(props: AppPropsWithLayout) {
```

Comment:
Any particular reason for this?

Reply:
I just noticed it was generic in the component inspector. Do you want it to stay the same?

Reply:
No, it's fine as far as I know it doesn't affect anything, was just wondering